server/models/HistoryEra.js

```diff
@@ -45,6 +47,18 @@ class HistoryEra {
     this.startAutoCleanup(cleanupInterval, this.db);
   }
 
+  loadDatabase(userId, dbName) {
+    let dbPath = `${config.dbPath}${userId}/`;
+
+    let db = new Datastore({
```

Comment:
This should be `const`, but let's just return this after setting `this.ready`.

Reply:
it was basically the same method as it was before, but for the `userId` :)